packages/base/SkeletonLoader/src/MediaSkeletonLoader/MediaSkeletonLoader.tsx

```diff
@@ -94,22 +94,26 @@ export const getAttributes = (props: React.PropsWithChildren<Props>) => {
     case 'icon':
       attributes = getIconAttributes(props)
       break
-    case 'avatar':
-      attributes = getAvatarAttributes(props)
-      break
     case 'image':
       attributes = getImageAttributes(props)
       break
+    case 'avatar':
+    default:
```

Comment:
```suggestion
    case undefined:
```

Also here, let's better explicitly check for `undefined` here to avoid this fallback kicking in unexpectedly if other variants are added.

Reply:
@Q-Man Fixed, thanks!